check/indentationcheck.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
waarom is deze filenaam geen camelcase?

Reply:
geen idee, kan ik wel doen. Zo staat hij ook in de NGS_Automated repo...